src/StaticWebAssetsSdk/Tasks/WriteImportMapToHtml.cs

```diff
@@ -42,9 +42,11 @@ public partial class WriteImportMapToHtml : Task
     // 2.group = fingerprint placeholder
     // 3.group = file extension
     // wrapped in quotes
-    private static readonly Regex _assetsRegex = new Regex(@"""([^""]+)(#\[\.{fingerprint}\])([^""]+)""");
+    internal static readonly Regex _assetsRegex = new Regex(@"""([^""]+)(#\[\.{fingerprint}\])([^""]+)""");
```

Comment:
Use [generator](https://learn.microsoft.com/en-us/dotnet/standard/base-types/regular-expression-source-generators) ?

Reply:
See unit tests

Reply:
Generator is not supported by net472 https://learn.microsoft.com/en-us/dotnet/api/system.text.regularexpressions.generatedregexattribute?view=net-9.0&viewFallbackFrom=netframework-4.7.2